### PR TITLE
final keyword

### LINKED ISSUE #129: Use `final` or not
Hi,

It could be interesting to add a section about the use of the `final` keyword.

Hereafter a nice blog post about that topic: https://ocramius.github.io/blog/when-to-declare-classes-final/

### PR DESCRIPTION
**Pull request by Spomky**
This PR is a proposal to fix #129.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1449,6 +1450,72 @@ $car->dump();
...
+```php
+interface Vehicule
+{
+    /**
+     * @return string The color of the vehicule
+     */
+    public function getColor();
+}
+
+final class Car interface Vehicule
```
Comment (mikemand) [inline README.md]: This should be `final class Car implements Vehicule`, right? And Vehicle doesn't have an extra U before the l.

Comment (Spomky) [reply README.md]: Correct

**Comment by TomasVotruba** [comment]
Like it, clear, short and right to the point :+1:  Thank you for your contribution.

Could you also link this post: https://ocramius.github.io/blog/when-to-declare-classes-final/ which explains all in more detailed way (if sb would be curious)?

**Comment by Spomky** [comment]
Hi @TomasVotruba,

I've just added the reference as suggested.

Regards.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**